docs/docs/customization/form.md

```diff
@@ -1,8 +1,8 @@
-Pathfora now allows for fully customized form fields for modules of type [form](/types/form) and [gate](/types/form). These input fields include text, texarea, select, checkbox, and radio button.
+Pathfora now allows for fully customized form fields for modules of type [form](../../types/form.md) and [gate](../../types/form.md). These input fields include text, texarea, select, checkbox, and radio button.
 
-Try out our [form schema builder](/customization/form_builder) to easily build forms with a drag and drop interface and output a `formElements` object for your module config.
+Try out our [form schema builder](./form_builder.md) to easily build forms with a drag and drop interface and output a `formElements` object for your module config.
```

Comment:
Is there a distinction on when we including the `.md` for some and not others? :thinking:

Reply:
My heuristic was whether the links worked correctly or not 😆. Being not very familiar with mkdocs, I didn't realize these are equivalent:

```
[form schema builder](./form_builder.md)
```
and
```
[form schema builder](../form_builder)
```

Similarly, from above in the same file:

```
[form](../types/form.md)
```
is equivalent to
```
[form](../../types/form)
```

I updated the handful of links still including `.md` to _not_ include it, adjusting the paths so the links would still work 🙃.